core/field_variable.js

```diff
@@ -138,6 +204,8 @@ Blockly.FieldVariable.prototype.setValue = function(value) {
  * @private
  */
 Blockly.FieldVariable.prototype.getVariableTypes_ = function() {
+  // TODO: Why does this happen every time, instead of once when the workspace
```

Comment:
Because we don't have a way to specify variable types ahead of time, so this list depends on which variables have been created so far. Part of issue #1513

Reply:
I'll change the TODO to poitn to #1513.